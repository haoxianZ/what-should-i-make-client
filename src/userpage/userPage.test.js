import React from 'react';
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import UserPage from './userPage'

describe(`UserPage component`, () => {

  it('renders user page', () => {
    const match={params:{userId:1}};
    const wrapper = shallow(<UserPage match={match}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
})