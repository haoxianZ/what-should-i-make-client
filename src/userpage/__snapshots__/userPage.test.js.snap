// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserPage component renders user page 1`] = `
<section
  className="userPage"
>
  <Link
    style={
      Object {
        "textDecoration": "none",
      }
    }
    to="/"
  >
    <h2>
      What Should I Make?
    </h2>
  </Link>
  <nav>
    Things in my fridge:
    <br />
  </nav>
  <form
    onSubmit={[Function]}
  >
    <label
      htmlFor="newNote"
    >
      Add ingredient to the fridge:  
    </label>
    <textarea
      id="newNote"
      name="newNote"
      required={true}
      type="textarea"
    />
    <button
      className="submitBtn"
      type="submit"
    >
      Add
    </button>
  </form>
  <div>
    <Display
      ingredients={Array []}
      user_id={1}
    />
  </div>
</section>
`;
